packages/ceramic-core/src/__tests__/ceramic.test.ts

```diff
@@ -48,12 +39,15 @@ describe('Ceramic integration', () => {
 
   beforeAll(async () => {
     tmpFolder = await tmp.dir({ unsafeCleanup: true })
-    ipfs1 = await Ipfs.create(genIpfsConf(tmpFolder.path, 0))
-    ipfs2 = await Ipfs.create(genIpfsConf(tmpFolder.path, 1))
-    ipfs3 = await Ipfs.create(genIpfsConf(tmpFolder.path, 2))
-    multaddr1 = (await ipfs1.id()).addresses[0].toString()
-    multaddr2 = (await ipfs2.id()).addresses[0].toString()
-    multaddr3 = (await ipfs3.id()).addresses[0].toString()
+    ipfs1 = await IpfsUtils.create(tmpFolder.path, 0)
```

Comment:
Seems like these calls could be made in parallel which should help run the tests faster, ex:
```ts
async function createIPFS(path, id): Promise<[Ipfs, string]> {
  const ipfs = await IpfsUtils.create(path, id)
  const ipfsID = await ipfs.id()
  return [ipfs, ipfsID.addresses[0].toString()]
}

await Promise.all([1, 2, 3].map(id => createIPFS(tmpFolder.path, id)))
```

Reply:
nice 👍

Reply:
yep, it can be done that way 👍